corehq/motech/repeaters/forms.py

```diff
@@ -21,6 +21,11 @@
 class GenericRepeaterForm(forms.Form):
 
     def __init__(self, *args, **kwargs):
+        self.repeater_name = ""
+        data = kwargs.get('data')
+        if data:
```

Comment:
```suggestion
        if data in kwargs:
```

Reply:
The next line (in the if branch) needs this data variable.
```
self.repeater_name = data.get("name")
```